meowrse.c

```diff
@@ -115,8 +177,20 @@ void displayHelpMessage() {
     printf("  meow-to-char    Translate meow to characters\n");
     printf("  morse-to-meow         Translate morse to meow\n");
     printf("  meow-to-morse         Translate meow to morse\n");
+    printf("  file-to-meow         Translate file to meow\n");
+    printf("  meow-to-file         Translate meow to file\n");
     printf("Options:\n");
     printf("  -h, --help        Display this help message\n");
+    printf("  -o, --output      Output to a file (file translation only)\n");
+}
+
+char* lower(char* s, int size) {
+    char* lower = calloc(size, sizeof(char));
+    for (int i = 0; i < size; i++) {
```

Comment:
recomendaria só assumir que o s é null terminated e:
```suggestion
char* lower(char* s) {
    size_t length = strlen(s);
    char* lower = calloc(length + 1, sizeof(char));
    for (int i = 0; i < length; i++) {
```

Reply:
ok